docs/Embedded/AssetSync.md

```diff
@@ -0,0 +1,14 @@
+---
+id: AssetSync
+title: Syncing Assets From Unity
+---
+
+_\[Note: Access to this feature requires a [Studio Subscription](https://enklu.com/pricing). [Contact us](/contact) for details.\]_
```

Comment:
Minor, but I'm usually not a fan of back to back links to different sources. Maybe just tweak it to "For more details _contact us_."?

Reply:
yep, agreed. and really, the contact us should link to the enklu.com contact us page